pages/billing.php

```diff
@@ -1,6 +1,6 @@
 <div class="pmpro_billing_wrap">
 <?php
-	global $wpdb, $current_user, $pmpro_msg, $pmpro_msgt, $show_paypal_link;
+	global $wpdb, $current_user, $pmpro_msg, $pmpro_msgt, $show_check_payment_instructions, $show_paypal_link;

```

Comment:
I know we don't prefix all of our globals, but let's prefix any new one like this.